Add vehicle and permit for order in extended permit

tonipel opened the pull request:
## Description

Allow searching for orders in admin ui with permit id. Add permit and vehicle to order when creating an extended permit.

## Context

[PV-820](https://helsinkisolutionoffice.atlassian.net/browse/PV-820)

## How Has This Been Tested?

Tested manually and with unit test.

## Manual Testing Instructions for Reviewers

When new extended permit has been created, in admin ui they should have permit id and vehicle information visible.
All orders should now be able to be searched with permit id.

## Screenshots

mhieta reviewed (changes requested):
PV-820 is all about rendering order list data for permit extension request-specific rows in Admin UI. So the issue is that the results, that are permit extension request-based orders, do not show permit id(s) and permit vehicle registration number(s) column data at all.

See the following screenshot:

![admin ui orders list view for extension requests](https://github.com/City-of-Helsinki/parking-permits/assets/2784933/f9be89bc-3588-46aa-b5ce-47ded0bd99ed)

Moreover, there is no need to directly alter the actual search behaviour.

tonipel commented:
This solution adds the permit id and registration number to orders when new order is created, so the data does not exist for older orders. However, I added a management command which populates those fields for orders that doesn't have such fields set.

Ticket mentions that orders should be searchable with permit id and registration number. Registration number search is already a functioning feature, but I added a search possibility with permit id as well in this PR.

I did not make changes yet to "Tunnustyyppi", since that has not been discussed before nor mentioned in the ticket, but I'll add that as well if needed.

mhieta reviewed (approved):
Ok, that new management command seemed to fix the issue for previous orders.
Therefore approved 👍
